src/pkg/retention/job.go

```diff
@@ -111,7 +111,7 @@ func (pj *Job) Run(ctx job.Context, params job.Parameters) error {
 	}
 
 	// Run the flow
-	results, err := processor.Process(allCandidates)
+	results, err := processor.Process(ctx.SystemContext(), allCandidates)
```

Comment:
Is the `ctx.SystemContext()` with a `ormer` pupoluated? Seems the `ctx` is passed to a DAO method in the final

Reply:
yup, the ctx.SystemContext() returns a context.Context with a ormer pupoluated.